### DISCUSSION TITLE
`log` MSRV update to 1.60

### DISCUSSION THREAD

**Discussion by dpc** [rust-bitcoin/rust-bitcoin#1798] [General]
https://www.reddit.com/r/rust/comments/12jepss/log_is_going_to_bump_msrv_to_160/

I immediately thought of `rust-bitcoin`, but I see that it doesn't seem to be using it. But you might want to be aware or contribute to the discussion about MSRV best practices.

**Comment by apoelstra** [comment]
Thanks for linking here. Judging by the majority of comments and the upvoting patterns it looks like it's full of web developers who never need their software to work and certainly don't need *software they wrote last year* to work. The stuff about "disrupting" industries with their YOLO attitudes gave me a chuckle, but I personally don't think it'd be a good use of my time to wade into it.

**Reply by TheBlueMatt** [reply]
Yea, I've tried engaging in those discussions repeatedly. At the end of the day its orders of magnitude simpler to just not take on any dependencies and NIH everything than to convince the rest of the rust community this stuff matters. Of course its also asking them to do (marginally) more work for our use-cases, and if they dont want to, that's up to them.

**Reply by Kixunil** [reply]
Unfortunately part of it is Rust moving slowly with adding some fundamental improvements. And that slowness is warranted since it avoids getting us into mess like C++ got into.

Anyway, I still believe forking > NIH.
